src/rules/at-conditional-rule-no-parentheses/README.md

```diff
@@ -0,0 +1,43 @@
+# at-conditional-rule-no-parentheses
```

Comment:
Maybe the name could be changed to `at-rule-conditional-no-parentheses`, so that it can be put under the `at-rule` category that many stylelint rules are using. What do you think @rambleraptor ?